cpp/src/io/parquet/parquet.hpp

```diff
@@ -57,6 +57,15 @@ struct TimeUnit {
   Type type;
 };
 
+struct DateUnit {
```

Comment:
Why are we inventing a non-existent parquet logical type? I can see this leading to much confusion in the future...at a minimum this should be documented. I see that you set the logical type to `DATE` ultimately and don't persist the `DateUnit`, but this should be made clear somewhere. Since there's already a precedent for adding to cudf specific fields to `SchemaElement`, perhaps this extra date info could go there instead?

Reply:
I agree. I initially added this for some semantic consistency with arrow which has both Date32 and Date64 types but eventually both are converted/hardcoded to Parquet's Date (aka Date32) type regardless so while this may make sense in cudf types (if and when we are supporting Date64), it doesn't make sense in Parquet. Reverting the change.